### DISCUSSION TITLE
User management function not working

### DISCUSSION THREAD

**Discussion by devansh-pan** [supabase/supabase#32027] [Questions] [answered]
The function not working when i update auth.users

**Comment by GaryAustin1** [comment, marked as answer]
You need to provide more details on what you mean and what not working means.  

I can guess you are talking about the example here: https://supabase.com/docs/guides/auth/managing-user-data?queryGroups=language&language=js#accessing-user-data-via-api

That is example code for inserting new information to a profile table when a user is created.  It does not handle updating but you would write a similar function with an update trigger if that is what "not working when I update" means.
